### ISSUE TITLE
--minimal-output for non test run

### ISSUE THREAD

**Issue by garyluu** [ShahradR/action-taskcat#90]
If I'm trying to use taskcat for anything other than "test run", it will fail because the --minimal-output is not recognized.  For example, I'm currently using the "lint" command and I get:

```
Installing collected packages: cfn-lint
  Attempting uninstall: cfn-lint
    Found existing installation: cfn-lint 0.35.0
    Uninstalling cfn-lint-0.35.0:
      Successfully uninstalled cfn-lint-0.35.0
Successfully installed cfn-lint-0.45.0
WARNING: You are using pip version 20.2.2; however, version 21.0.1 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
 _            _             _   
| |_ __ _ ___| | _____ __ _| |_ 
| __/ _` / __| |/ / __/ _` | __|
| || (_| \__ \   < (_| (_| | |_ 
 \__\__,_|___/_|\_\___\__,_|\__|
                                


version 0.9.23
usage: taskcat [args] <command> [args] [subcommand] [args] 
taskcat: error: unrecognized arguments: --minimal-output
```

https://github.com/ShahradR/action-taskcat/blob/main/entrypoint.sh#L11

**Comment by ShahradR** [comment]
Thank you for flagging this, really appreciated!

I just pushed release [v1.3.2](https://github.com/ShahradR/action-taskcat/releases/tag/v1.3.2) which removes the `--minimal-output` flag. It was added because the `test run` output prints out of order, but I forgot to check whether it impacted other commands.

I'll be sure to add some tests running against other commands to prevent another regression like this—I hope this didn't affect your workflows too much!

Take a look, see if the new release fixes your issue?
